extensions/adobe/experience/target/experienceevent.schema.json

```diff
@@ -58,6 +58,12 @@
           "items": {
             "$ref": "https://ns.adobe.com/experience/target/activity"
           }
+        },
+        "https://ns.adobe.com/experience/target/supplementalDataID": {
```

Comment:
@cluby-adobe @vazdauta  - Can you explain what this ID actually does please? I don't have enough context on what to do here. I want to get this closed in the next day or two please.

Reply:
Next hour or two if we want this to make 0.9.6 :)

Reply:
My research shows that this is mostly documented as a Target related feature. It is not documented in the core Analytics implementation documentation.

I am leaning to keeping this under Target. @cluby-adobe is this a blocking issue for you?

Reply:
It's used as a key to merge multiple ExperienceEvents together as one downstream.  It's populated by Analytics, Target, and AdCloud is in the works.  Currently the only downstream merger is Analytics' current data collection pipeline but in the Platform world we'll have multiple.

It for sure doesn't belong in the Target space and since it's populated by multiple solutions I don't think it belongs under any solution.  I don't think I would put it in the core at this point because it's specific to Adobe Experience Cloud.  So I think putting it under the adobe/experience namespace - https://github.com/adobe/xdm/blob/master/extensions/adobe/experience/experienceevent.schema.json - is the right place to put it.

Reply:
I like that idea, my only concern is that we would really like to get rid of that "union" schema at some point.  I guess we could introduce a common "base event" for all the solutions when we do that refactor.

For now I am leaning towards just putting it onto the core experience event, but we would need to have a better name and more general description.  I believe what we are saying is that sometimes a single action or activity results in multiple events, and they need to be annotated with a common value to link them all back together.  Can we call this "actionID" or maybe a "eventLinkingID" or similar?

Reply:
@kstreeter 
+1 on eventLinkingID for the final standard solution, but that is in the future. 
 I think we need to have a larger discussion in source event identity, Launch and a number of other things before we can make a decision on the standard. I bet we get it wrong if we do anything in the short time we have now. And we will still need the sdid extension anyway.

Reply:
@cdegroot-adobe ,why would we need SDID as well if that same value was carried in another field?